Assets/Scripts/SS3D/Systems/Entities/Humanoid/Ragdoll.cs

```diff
@@ -0,0 +1,334 @@
+using FishNet.Component.Animating;
+using FishNet.Object;
+using FishNet.Object.Synchronizing;
+using System;
+using System.Linq;
+using UnityEngine;
+
+namespace SS3D.Systems.Entities.Humanoid
+{
+    [RequireComponent(typeof(Animator), typeof(NetworkAnimator), typeof(CharacterController))]
+    [RequireComponent(typeof(HumanoidLivingController))]
+	public class Ragdoll : NetworkBehaviour
+	{
+        [SerializeField]
+		private Transform _armatureRoot;
+        private Transform _hips;
+        private Transform _character;
+		private Animator _animator;
```

Comment:
weird tabulation, put spaces only, no tabs. You should be able to do that in your IDE automatically for the whole file.